etl/xml_importer/xpaths.py

```diff
@@ -143,9 +145,16 @@
     "Resource_LinkResource_Path": 'lido:resourceRepresentation/lido:linkResource',
     "Resource_Photographer_Path": 'lido:resourceSource[@lido:type="Fotograf"]/lido:legalBodyName/lido:appellationValue',
 
+
+
     #RIGHTS
     "Rights_Type_Path": "lido:rightsType/lido:conceptID",
     "Rights_Holder_Path": "lido:rightsHolder/lido:legalBodyID",
     #rightsHolder
+
+    #ENTITY
+    ##Prios sind niedrig oder nicht vorhanden, daher erstmal nicht beachten
+
+    "namespace": "{'lido': 'http://www.lido-schema.org'}"
```

Comment:
Remove unneeded comments and `"namespace"` key from paths dictionary

Reply:
unneeded comments  is deleted